tests/run-make/core-no-oom-handling/rmake.rs

```diff
@@ -0,0 +1,16 @@
+// This test checks that the core library can still compile correctly
+// when the no_global_oom_handling feature is turned on.
```

Comment:
Nit: it only tests that *compilation succeeds*, it does not check for *correctness*.

Reply:
I wrote that exact "correctly" terminology in a couple of similar tests to this one, so I have changed them as well. Thanks!